bican.mccarroll.utils/DESCRIPTION

```diff
@@ -0,0 +1,16 @@
+Package: bican.mccarroll.utils
+Title: What the Package Does (One Line, Title Case)
+Version: 0.0.0.9000
+Authors@R: 
+    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"))
+Description: What the package does (one paragraph).
+License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
```

Comment:
Look at the ./R/bican.mccarroll.helloworld/DESCRIPTION for an example of how to do the license.  You'll also need  LICENSE file (you can copy the one from ./R/bican.mccarroll.helloworld/LICENSE)